ex2/ex2.js

```diff
@@ -8,3 +8,25 @@
  *     fn(",,,", 4) === -1
  *     fn("hel,lowo,rld", 0) - Throws
  */
+function getTheCommaIndexAt(str, val) {
+	var index = -1, returnVal = 'Error';
```

Comment:
I didn't mean to return "Error". Read about Error at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error and https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/throw.